src/request.js

```diff
@@ -0,0 +1,237 @@
+let http = require('http');
+let https = require('https');
+let assert = require('assert');
+let urllib = require('url');
```

Comment:
hahahah, it's funny because i specifically picked that name because it's what python uses :P